src/app/hocs/PrivateRoute.tsx

```diff
@@ -17,11 +19,11 @@ export const PrivateRoute: React.SFC<PrivateRouteProps> = (props) => {
   } = props;
 
   if (isFetching) {
-    return null;
+    return <div className="SplashScreen SplashScreen-whiteScreen" />;
   }
 
   if (!isSubscribing) {
-    history.replace('/' + window.location.search);
+    history.replace(`${RoutePaths.HOME}/${window.location.search}`);
```

Comment:
드물지만 `window.location`과 라우터의 `location`이 일치하지 않는 경우가 있습니다. `withRouter`를 붙인 뒤 이렇게 쓰는 게 좋겠네요.

```suggestion
    props.history.replace({
      pathname: RoutePaths.HOME,
      search: props.location.search,
    });
```